packages/tooling/fast-tooling-wasm/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+
+const appDir = path.resolve(__dirname, "./app");
+const outDir = path.resolve(__dirname, "./www");
+
+module.exports = {
+    devServer: {
+        compress: false,
+        historyApiFallback: true,
```

Comment:
I'm curious (has nothing to do with this PR).  Is there a way to include "historyApiFalback" in the production webpack build such that we would not need to install this dependency as a node module in production?

Reply:
I don't think so because that's a sever configuration, if you were to boot up the prod webpacked version with `express` or what have you, you would add it there.